src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -1337,6 +1337,40 @@ namespace tools::wallet_rpc {
     };
   };
 
+  OXEN_RPC_DOC_INTROSPECT
+  // Export transfers to csv
+  struct EXPORT_TRANSFERS : RPC_COMMAND
+  {
+    static constexpr auto names() { return NAMES("export_transfers"); }
+
+    struct request
+    {
+      std::string filename; // String for the filename of the CSV to write
```

Comment:
I'm thinking we should drop the filename argument, and simply return the CSV contents via RPC as a (big) string.  That would then push the responsibility for prompting for a filename, potentially overwriting, etc. to the GUI wallet which is better able to handle that sort of thing than the rpc wallet.